src/views/createprojectmodal/components/NetworkTab.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { FC } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import {
+  Alert,
+  AlertVariant,
+  DropdownItem,
+  FormGroup,
+  Text,
+  TextInput,
+} from '@patternfly/react-core';
+import Select from '@utils/components/Select/Select';
+
+import { CreateProjectModalFormState, NETWORK_TYPE, networkTypeLabels } from '../constants';
+
+const NetworkTab: FC = () => {
+  const { t } = useTranslation();
+  const { control, register, setValue, watch } = useFormContext<CreateProjectModalFormState>();
+
+  const networkType = watch('networkType');
+
+  return (
+    <div className="create-project-modal__networktab">
+      <Alert
+        className="create-project-modal__network-alert"
+        isInline
+        title={t(
+          'Define the network used by VirtualMachines and Pods to communicate in this project',
+        )}
+        variant={AlertVariant.info}
+      >
+        <Text>
+          {t('This network must be created before you create any workload in this project')}
+        </Text>
+      </Alert>
+
+      <Controller
+        control={control}
+        name="networkType"
+        render={({ field: { onChange, value } }) => (
+          <FormGroup fieldId="network-type" isRequired label={t('Network type')}>
+            <Select
+              id="network-type"
+              selected={value || NETWORK_TYPE.POD_NETWORK}
+              toggleContent={networkTypeLabels[value || NETWORK_TYPE.POD_NETWORK]}
+            >
+              <>
+                {Object.entries(networkTypeLabels)
+                  // .filter(([type]) =>
+                  //   !isAdmin && type === NETWORK_TYPE.CLUSTER_UDN.toString() ? false : true,
+                  // )
```

Comment:
?

Reply:
This is becuase cluster UDN is not ready. When it will be ready we just need to de-comment those